Xavier/install_php.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -x
+
+# Install PHP
+# ========================
+#sudo apt-get install php-fpm php-common php-curl php-gd php-cli php-mysql php-mbstring php-xml
+sudo apt-get install -y php8.2-fpm
+
+# Enable PHP via FPM : FastCGI Process Manager
+sudo a2enmod proxy_fcgi setenvif
+sudo a2enconf php8.2-fpm
+
+# Create sample PHP file
+# ========================
+html_root="/var/www/testapache"
+mkdir -p $html_root
+
+cat > "$html_root"/index.php <<-EOF
+<?php
+    print("Hello World");
```

Comment:
On nous a dit qu'il ne valait mieux pas fermer la balise PHP, pour éviter d'envoyer des espaces après la balise fermante :

> If a file is purely PHP, it might end without a closing ?> tag to avoid accidental white space or new lines being sent to the browser, which can cause issues especially when dealing with headers or session management. For This practice helps prevent unintended output which can interfere with functions that modify HTTP headers, such as header() or setcookie().

Reply:
See : 
https://www.php.net/manual/en/language.basic-syntax.phptags.php
>If a file contains only PHP code, it is preferable to omit the PHP closing tag at the end of the file. This prevents accidental whitespace or new lines being added after the PHP closing tag, which may cause unwanted effects because PHP will start output buffering when there is no intention from the programmer to send any output at that point in the script. 

See also :
https://stackoverflow.com/questions/4410704/why-would-one-omit-the-close-tag
https://stackoverflow.com/questions/236493/what-are-the-arguments-in-favor-of-php-closing-tags-for-php-only-files

Reply:
Je ne savais pas, merci !